src/main/utils/__tests__/windowStateKeeper.test.ts

```diff
@@ -2,6 +2,11 @@
 
 import windowStateKeeper from '../windowStateKeeper'
 
+jest.mock('../../utils/metricsLogger.ts', () => ({
+  __esModule: true,
+  default: jest.fn(() => {})
+}))
+
```

Comment:
I think you just need a 
```suggestion
   expect(metricsLogger).toHaveBeenCalledTimes(1)
    expect(metricsLogger).toHaveBeenCalledWith({
      eventType: 'WindowSizePreferences',
      data: {
        windowStateInfo: <some payload>
      }
    })
```
as an assertion in the test